patterns/dswp-vertical-card.php

```diff
@@ -0,0 +1,133 @@
+            <div class="wp-block-column" style="border-style:none;border-width:0px;padding-top:var(--wp--preset--spacing--10);padding-right:0rem;padding-bottom:var(--wp--preset--spacing--10);padding-left:0rem"><!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"0","left":"0"}},"border":{"radius":"0px","color":"#d9d9d9","width":"1px"}},"className":"dswp-vertical-card","layout":{"type":"constrained","contentSize":"111px","justifyContent":"left","wideSize":""}} -->
+                <div class="wp-block-group alignfull dswp-vertical-card has-border-color" style="border-color:#d9d9d9;border-width:1px;border-radius:0px;padding-right:0;padding-left:0"><!-- wp:image {"align":"full","id":15402,"sizeSlug":"full","linkDestination":"none","className":"size-full"} -->
+                    <figure class="wp-block-image alignfull size-full"><img src="<?php echo esc_url( get_template_directory_uri() . '/assets/images/square-256.png' ); ?>" alt="" /></figure>
+                    <!-- /wp:image -->
+
+                    <!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|40","left":"var:preset|spacing|40"}}},"layout":{"type":"constrained"}} -->
+                    <div class="wp-block-group alignfull" style="padding-right:var(--wp--preset--spacing--40);padding-left:var(--wp--preset--spacing--40)"><!-- wp:heading {"textAlign":"left","level":5,"style":{"elements":{"link":{"color":{"text":"var:preset|color|typography-primary-color"}}},"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"var:preset|spacing|50","bottom":"0"}}},"textColor":"typography-primary-color","className":"wp-block-heading is-style-default","fontSize":"xx-large"} -->
+                        <h5 class="wp-block-heading has-text-align-left is-style-default has-typography-primary-color-color has-text-color has-link-color has-xx-large-font-size" style="margin-top:var(--wp--preset--spacing--50);margin-bottom:0;padding-top:0;padding-bottom:0">Neque porro quisquam</h5>
+                        <!-- /wp:heading -->
+
+                        <!-- wp:paragraph {"style":{"spacing":{"margin":{"top":"var:preset|spacing|20","bottom":"var:preset|spacing|40","left":"0","right":"0"},"padding":{"right":"0","left":"0"}},"typography":{"lineHeight":"1.8"}}} -->
+                        <p style="margin-top:var(--wp--preset--spacing--20);margin-right:0;margin-bottom:var(--wp--preset--spacing--40);margin-left:0;padding-right:0;padding-left:0;line-height:1.8">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus molestie a mi eu dictum. Sed eget dictum massa.</p>
+                        <!-- /wp:paragraph -->
+
+                        <!-- wp:paragraph {"style":{"spacing":{"margin":{"top":"0","bottom":"0"}}}} -->
+                        <p style="margin-top:0;margin-bottom:0"><a href="https://localhost/ticorp/test2/" data-type="page" data-id="15259">Neque porro quisquam</a></p>
```

Comment:
```suggestion
                        <p style="margin-top:0;margin-bottom:0"><a href="#example" data-type="page" data-id="15259">Neque porro quisquam</a></p>
```
Might be better to replace the links with something like this so at least if the user forgets to set the href it doesn't try to go to localhost.

Reply:
done, please re-approve